sql-statements/sql-statement-admin-show-ddl.md

```diff
@@ -0,0 +1,114 @@
+1 row in set (0.00 sec)
+```
+
+### `ADMIN SHOW DDL JOBS`
+
+To view all the results in the current DDL job queue (including tasks that are running and waiting to be run) and the last ten results in the completed DDL job queue, use `ADMIN SHOW DDL JOBS`:
+
+{{< copyable "sql" >}}
+
+```sql
+ADMIN SHOW DDL JOBS;
+```
+
+```
+mysql> ADMIN SHOW DDL JOBS;
++--------+---------+-------------------------+--------------------------+--------------+-----------+----------+-----------+---------------------+---------------------+--------+
+| JOB_ID | DB_NAME | TABLE_NAME              | JOB_TYPE                 | SCHEMA_STATE | SCHEMA_ID | TABLE_ID | ROW_COUNT | START_TIME          | END_TIME            | STATE  |
++--------+---------+-------------------------+--------------------------+--------------+-----------+----------+-----------+---------------------+---------------------+--------+
+|     51 | test    | t1                      | create table             | public       |         1 |       50 |         0 | 2020-08-09 15:02:43 | 2020-08-09 15:02:43 | synced |
+|     49 | ontime  | ontime                  | rebase auto_increment ID | public       |        45 |       47 |         0 | 2020-08-09 13:41:53 | 2020-08-09 13:41:53 | synced |
+|     48 | ontime  | ontime                  | create table             | public       |        45 |       47 |         0 | 2020-08-09 13:28:23 | 2020-08-09 13:28:23 | synced |
+|     46 | ontime  |                         | create schema            | public       |        45 |        0 |         0 | 2020-08-09 13:28:22 | 2020-08-09 13:28:22 | synced |
+|     44 | mysql   | opt_rule_blacklist      | create table             | public       |         3 |       43 |         0 | 2020-08-09 10:17:18 | 2020-08-09 10:17:18 | synced |
+|     42 | mysql   | expr_pushdown_blacklist | create table             | public       |         3 |       41 |         0 | 2020-08-09 10:17:17 | 2020-08-09 10:17:18 | synced |
+|     40 | mysql   | stats_top_n             | create table             | public       |         3 |       39 |         0 | 2020-08-09 10:17:17 | 2020-08-09 10:17:17 | synced |
+|     38 | mysql   | bind_info               | create table             | public       |         3 |       37 |         0 | 2020-08-09 10:17:17 | 2020-08-09 10:17:17 | synced |
+|     36 | mysql   | default_roles           | create table             | public       |         3 |       35 |         0 | 2020-08-09 10:17:17 | 2020-08-09 10:17:17 | synced |
+|     34 | mysql   | role_edges              | create table             | public       |         3 |       33 |         0 | 2020-08-09 10:17:17 | 2020-08-09 10:17:17 | synced |
++--------+---------+-------------------------+--------------------------+--------------+-----------+----------+-----------+---------------------+---------------------+--------+
+10 rows in set (0.01 sec)
+```
+
+You may limit the number of rows shown by specifying a number and where condition. i.e.
```

Comment:
how about describing more on how to read the output of `SCHEMA_STATE`?

Reply:
I have added a better example in https://github.com/pingcap/docs/pull/3587/commits/d4bd51f4aa280db62117b6d494e6182c3de6a767

PTAL, thx :-)